rules/windows/persistence_user_account_added_to_privileged_group_ad.toml

```
@@ -15,6 +15,37 @@ index = ["winlogbeat-*", "logs-windows.*", "logs-system.*"]
 language = "eql"
 license = "Elastic License v2"
 name = "User Added to Privileged Group in Active Directory"
+note = """## Triage and analysis
+
+### Investigating User Added to Privileged Group in Active Directory
+
+Privileged accounts and groups in Active Directory are those to which powerful rights, privileges, and permissions are
+granted that allow them to perform nearly any action in Active Directory and on domain-joined systems.
```

This is a near duplicate of the description. We should probably revise more.

```suggestion
Privileged accounts and groups in Active Directory have powerful rights, privileges, and permissions
granted, which allow them to perform nearly any action in Active Directory and on domain-joined systems.
```

---

That is adapted the official description from ms docs
https://docs.microsoft.com/en-us/windows-server/identity/ad-ds/plan/security-best-practices/appendix-b--privileged-accounts-and-groups-in-active-directory#appendix-b-privileged-accounts-and-groups-in-active-directory-1

I see no problem in similar texts in the description, as this is basically another description of the same thing.